coderedcms/project_template/basic/project_name/settings/base.py

```diff
@@ -203,3 +203,7 @@
 # Tags
 
 TAGGIT_CASE_INSENSITIVE = True
+
+# Sets default for primary key IDs - change to BigAutoField if you prefer a 64-bit integar instead
+# See https://docs.djangoproject.com/en/3.2/ref/models/fields/#autofield
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
It looks like as of Django 3.2, BigAutoField is the default, therefore it would make sense that since this template is for brand new projects, we should follow their default.

https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-DEFAULT_AUTO_FIELD